qa/tasks/quiescer.py

```diff
@@ -0,0 +1,269 @@
+"""
+Thrash mds by simulating failures
+"""
+import logging
+import contextlib
+
+from teuthology import misc as teuthology
+
+from tasks import ceph_manager
+from tasks.cephfs.filesystem import MDSCluster, Filesystem
+from tasks.thrasher import ThrasherGreenlet
+
+import random
+import math
+import errno
+import json
+import time
+
+from io import StringIO
+
+log = logging.getLogger(__name__)
+
+class Quiescer(ThrasherGreenlet):
+    """
+    The Quiescer does periodic quiescing of the configured paths, by default - the root '/'.
+
+    quiesce_timeout: maximum time in seconds to wait for the quiesce to succeed
+    quiesce_factor: the percentage of time we allow the system to stay quiesced
```

Comment:
percent of what

Reply:
percent of total run time that the system was quiesced. 0.5 means 50% means that the system spends as much time quiesced as running. 0.1 means that for every minute quiesced the system will run 9 minutes freely